streams/src/main/java/org/apache/kafka/streams/kstream/internals/KStreamImpl.java

```diff
@@ -93,8 +92,7 @@
 
     private static final String WINDOWED_NAME = "KSTREAM-WINDOWED-";
 
-    private static final LongSerializer LONG_SERIALIZER = new LongSerializer();
-    private static final LongDeserializer LONG_DESERIALIZER = new LongDeserializer();
+    private static final SerDe<Long> LONG_SER_DE = SerDes.LONG();
```

Comment:
@ijuma: I think perhaps `SerDes.LONG()` (and its variants for other types) always returns a new instance of the corresponding serde.  Perhaps we should, as you mentioned previously, return a cached instance instead of always creating a new instance (think: `lazy val`).